public/scripts/worktrollyUI.js

```diff
@@ -9,6 +9,12 @@ $("#startNewSprint").ready(function() {
     $("#startNewSprint").hide(0);
 });
 
+$("#logWorkPage").click(function() {
+    newPage = "logWorkTask";
+    uiLoader();
+    fillDataIntoLogWorkPage();
```

Comment:
Basic Programming skills needed to be improved, This method is taking arguments which you are not sending.

Please test the code before sending for code review.
It's a kid job. Please please please focus.

Reply:
ok